### DISCUSSION TITLE
Raspberry Pi OS?

### DISCUSSION THREAD

**Discussion by AaronD-GH** [waydroid/waydroid#733] [General]
Not sure which category this really goes in, so I put it here.

Anyway, I have a Raspberry Pi 4 with the standard RPi OS on it.  Updated regularly.  I've seen several ways to completely replace that with either a version of Android or an open-source equivalent, but that's not what I'm looking for.  I want to keep the Raspberry Pi OS as-is, with all of the apps that I already have, and also put some Android apps on it too, that I like from my phone.

I ran the repo script from the [Install Instructions](https://docs.waydro.id/usage/install-on-desktops)
`curl https://repo.waydro.id | sudo bash`
and it said
`[!] Distribution "buster" is not supported`

Does this project run on the fully up-to-date standard Pi OS?  If so, how to do it?

**Comment by PhilDevProg** [comment]
This is weird... Raspberry Pi OS should use Debian 11 Bullseye but you seem to use a Raspberry Pi Os based on Debian 10 Buster. Can you please run `lsb_release -a`?

**Comment by AaronD-GH** [comment]
It's old, originally ran for years on a Pi 3B+.  Then I dd-cloned the card to preserve the original, and put the "new" one in a Pi 4.  Ran just fine, so I kept it.  Even that was several years ago.  It still gets updates.  (`sudo apt {update|full-upgrade|autoremove}`, then reboot)
```
aaron@Aaron-NotesPC:~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
aaron@Aaron-NotesPC:~ $ 
```
I guess I was slightly confused about the "latest" version.  I figured if it still gets updates and doesn't tell me about another one, then it must be current.  Apparently not.

**Comment by MinGW32-2006** [comment]
Run waydroid on an 1GB RAM device seems impossible. You can update your RPi OS to bullseye and try anyway.

**Comment by gitmanbloggus** [comment]
I have a rpi4 with 8gb ram running Ubuntu, is that enough? It's running Ubuntu 23.10
